libcore/FileOperations/DialogUtils.vala

```diff
@@ -0,0 +1,181 @@
+namespace PF {
```

Comment:
I'm actually wondering what is this class and where did this come from? Because these functions seem to exist already in libcore/FileOperations/CommonJob.vala